### ISSUE TITLE
Missing API endpoint

### ISSUE THREAD

**Issue by jbwl** [guilhermesad/rspotify#161]
I would like to use the Spotify API endpoint

````
PUT https://api.spotify.com/v1/users/{user_id}/playlists/{playlist_id}/images
````
documented here: https://developer.spotify.com/web-api/upload-a-custom-playlist-cover-image/

But it's missing in rspotify. So it's not "full API endpoint coverage". Are there plans to include it?

**Comment by guilhermesad** [comment]
Hi @jbwl, indeed this endpoint was missing, thanks for noticing it. I've just [added it](https://github.com/guilhermesad/rspotify/commit/9bdcac859665169933dd74f825f44e6285241276), it's included in version `2.1.0`. Thanks!

**Comment by jbwl** [comment]
Wow, thank you very much for the quick addition!
